src/aiai_eval/utils.py

```diff
@@ -203,22 +207,39 @@ def check_supertask(architectures: Sequence[str], supertask: str):
             The supertask associated to a task written in kebab-case, e.g.,
             text-classification.
 
+    Returns:
+        tuple of list of str:
+            The architectures that are valid for the supertask, and the provided model architectures that
+            which are allowed for the supertask.
+
     Raises:
         InvalidArchitectureForTask:
             If the PascalCase version of the supertask is not found in any of the
             architectures.
     """
-    # Create boolean variable that checks if the supertask exists among the available
-    # architectures
-    supertask_is_an_architecture = any(
-        kebab_to_pascal(supertask) in architecture for architecture in architectures
-    )
+    # Create list which contains the supertask if it exists among the available architectures
+    supertask_which_is_architectures = [
+        supertask
+        for architecture in architectures
+        if kebab_to_pascal(supertask) in architecture
+    ]
```

Comment:
This list will always have exactly 0 or 1 element, which seems a bit silly. Why not use the previous Boolean value?